docs/team/skills.md

```diff
@@ -0,0 +1,103 @@
+# Skills and ways to contribute
+
+There are many ways that you can contribute to the JupyterHub / Binder projects.
+This page attempts to provide some pathways for community members to learn and
+apply their skills to the team.
+
+## Where to start?
+
+The short answer to this question is: it depends :-)  there are many
+different ways to contribute to the JupyterHub community - technical, mentorship,
+communications, community work, and documentation are just a few. You can either
+apply your skills to help out, or use it as an opportunity to learn something new!
+
+## A few general ways to help out
+
+* Look through the issues in a repository, and help respond to issues that haven't been addressed already
+* Guide conversation in issues so that they get closer to having a clear, actionable path forward
+* For general questions about using JupyterHub, assist the person and then improve the documentation
+  so that the answer is easier to find.
```

Comment:
Should we say something about guiding people to Discourse for help?

Reply:
Great suggestion @manics. Add a link here to the Discourse.